public/supabase_broker/squashed.sql

```diff
@@ -419,13 +435,21 @@ $$;
 
 ALTER FUNCTION "toit_artemis"."insert_pod"("_pod_id" "uuid", "_pod_description_id" bigint) OWNER TO "postgres";
 
-CREATE OR REPLACE FUNCTION "toit_artemis"."max_event_age"() RETURNS interval
+CREATE OR REPLACE FUNCTION "toit_artemis"."max_events"() RETURNS integer
+    LANGUAGE "sql" IMMUTABLE
+    AS $$
+    SELECT 100;
```

Comment:
Can we make this a proper number like 128 (power of two) or 101 (a prime)?

Reply:
done.